server/paymentProviders/opencollective/host.js

```diff
@@ -16,6 +18,24 @@ paymentMethodProvider.getBalance = () => {
   return Promise.resolve(maxInteger);
 };
 
+paymentMethodProvider.createPlatformTipTransaction = async payload => {
+  const transaction = payload.transaction;
+  const TransactionGroup = transaction.TransactionGroup || uuid();
+  const donationTransaction = {
+    ...transaction,
+    amount: transaction.data.platformTipInHostCurrency,
+    description: 'Financial contribution (Platform Tip) to Open Collective',
+    netAmountInCollectiveCurrency: transaction.data.platformTipInHostCurrency,
+    FromCollectiveId: payload.FromCollectiveId,
+    hostCurrencyFxRate: 1,
+    TransactionGroup,
+    PlatformTipForTransactionGroup: TransactionGroup,
```

Comment:
We're [changing](https://github.com/opencollective/opencollective-api/pull/5900) how things are working here: 
- `TransactionGroup` should have the same `TransactionGroup` than original transactions (added funds)
- `PlatformTipForTransactionGroup` does not exists anymore

Reply:
Done. 👍🏽